lib/saml/url_service.rb

```diff
@@ -53,10 +54,6 @@ def idme_url
       build_sso_url(LOA::IDME_LOA1)
     end
 
-    def signup_url
```

Comment:
Got rid of this because this is just `idme_url` with the query param appended. Now that we pass certain query params along from the controller, this specific method won't be necessary.